src/main/java/org/folio/edge/core/security/EphemeralStore.java

```diff
@@ -27,8 +27,10 @@ public EphemeralStore(Properties properties) {
       String tenants = properties.getProperty(PROP_TENANTS);
       if (tenants != null) {
         for (String tenant : COMMA.split(tenants)) {
-          String pass = properties.getProperty(tenant);
-          put(tenant, tenant, pass == null ? "" : pass);
+          String[] credentials = properties.getProperty(tenant).split(",");
```

Comment:
might want to code defensively here... what if the property is null?  We'd get an ArrayIndexOutOfBoundException

Reply:
Hi @craigmcnally ,
I can propose the following solution. Please, review and let me know if needed.

Thanks,
Slava